docs/introduction/record_agent.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: "contents"
+title: Recording Agents
+---
+
+# Recording Agents
+
+```{eval-rst}
+ .. py:currentmodule: gymnasium.wrappers
+
+During and after training an agnt, it is interesting to record how the agent acts over an episode, in particular to video the agent behaviour and log the total reward for each episode. This can be achieved through two wrappers: :class:`RecordEpisodeStatistics` and :class:`RecordVideo`, the first will track episode data such as the total rewards, episode length and time taken with the second saving the environment rendering as mp4 videos.
```

Comment:
agent*

It may be interesting to record agent behaviour over an episode and log the total reward accumulated. This can be achieved through two...